components/virtual-node/virtual-node-write-service-impl/src/main/kotlin/net/corda/virtualnode/write/db/impl/writer/DbConnection.kt

```diff
@@ -2,30 +2,31 @@ package net.corda.virtualnode.write.db.impl.writer
 
 import net.corda.db.core.DbPrivilege
 import net.corda.libs.configuration.SmartConfig
-import net.corda.schema.configuration.DatabaseConfig.DB_PASS
-import net.corda.schema.configuration.DatabaseConfig.DB_USER
 
 /**
- * Virtual node DB connection data.
+ * Represents a Virtual Node DB connection
  *
  * @property name Connection name
  * @property privilege Connection's DB privilege
  * @property config Connection configuration
  * @property description Connection's description
  */
-class DbConnection(val name: String, val privilege: DbPrivilege, val config: SmartConfig, val description: String) {
+
+interface DbConnection {
```

Comment:
```suggestion
internal interface DbConnection {
```
just to highlight the fact that it is not going to be used outside of this module.